tests/io.rs

```diff
@@ -1,5 +1,6 @@
 use std::collections::HashMap;
 use std::io::{ErrorKind, Read, Write};
+use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::{Arc, Mutex};
```

Comment:
```suggestion
use std::sync::Arc;
```
This import isn't used.